superset-frontend/src/views/CRUD/data/database/DatabaseModal/styles.ts

```diff
@@ -123,9 +124,14 @@ export const antDModalStyles = (theme: SupersetTheme) => css`
     color: ${theme.colors.grayscale.dark1};
     opacity: 1;
   }
+
   .ant-modal-title > h4 {
     font-weight: bold;
   }
+
+  .ant-modal-body {
+    height: ${theme.gridUnit * 180.5}px;
```

Comment:
will this work with the height constants above? Does it need to be a fixed number for those?

Reply:
I believe that since the body is long enough, it doesn't interfere with the form constants. The forms seem to appear properly when I run through them. Would it be better to store the body's height in a const like the form heights?